src/containers/Application/ApplicationHome.tsx

```diff
@@ -1,21 +1,87 @@
-import React from 'react'
-import { FullStructure } from '../../utils/types'
+import React, { useEffect } from 'react'
+import { Button, Divider, Header, Message, Segment, Sticky } from 'semantic-ui-react'
+import { FullStructure, SectionAndPage, StageAndStatus, TemplateDetails } from '../../utils/types'
 import useGetFullApplicationStructure from '../../utils/hooks/useGetFullApplicationStructure'
+import { ApplicationHeader, Loading } from '../../components'
+import strings from '../../utils/constants'
+import { useUserState } from '../../contexts/UserState'
+import SectionsProgress from '../../components/Sections/SectionsProgress'
+import { useRouter } from '../../utils/hooks/useRouter'
+import { ApplicationStatus } from '../../utils/generated/graphql'
 
 interface ApplicationProps {
   structure: FullStructure
+  template: TemplateDetails
 }
 
-const ApplicationHome: React.FC<ApplicationProps> = ({ structure }) => {
+const ApplicationHome: React.FC<ApplicationProps> = ({ structure, template }) => {
+  const {
+    query: { serialNumber },
+    push,
+    replace,
+  } = useRouter()
+  const {
+    userState: { currentUser },
+  } = useUserState()
+
   const { error, fullStructure } = useGetFullApplicationStructure({
     structure,
   })
 
-  console.log('FULL STRUCTURE', fullStructure)
-  console.log('responsesByCode', fullStructure?.responsesByCode)
-  console.log('isLoading', !fullStructure)
+  useEffect(() => {
+    if (!fullStructure) return
+    const { status } = fullStructure.info.current as StageAndStatus
+    if (status !== ApplicationStatus.Draft && status !== ApplicationStatus.ChangesRequired)
+      replace(`/application/${serialNumber}/summary`)
```

Comment:
`/summaryNEW`?

Reply:
Also -- why `replace` rather than `push`?

Reply:
I was just keep the same behaviour as before. But if pushing is better (to allow user going back to previous page) I can change to that. Thanks for the suggestion